test/tools/spec-runner/index.js

```diff
@@ -86,6 +86,14 @@ function translateClientOptions(options) {
         }
 
         options.autoEncryption.kmsProviders = kmsProviders;
+
+        if (options.autoEncryption.encryptedFieldsMap) {
+          const map = options.autoEncryption.encryptedFieldsMap;
+          if (map['default.encryptedCollection']) {
+            /* eslint no-console: 0 */
+            console.log(map['default.encryptedCollection'].fields[0]);
```

Comment:
Is this intentionally left in? If yes, it might be good to include some sort of fixed string in the log line, so that easy to dig up where the log output is coming from :)